astropy/cosmology/core.py

```diff
@@ -230,6 +339,7 @@ class Cosmology(metaclass=abc.ABCMeta):
 
     # Parameters
     __parameters__ = ()
+    __all_parameters__ = ()
```

Comment:
It would be nice to have only attribute tracking the parameters, but filtering for ``fixed=False`` every time we need ``__parameters__`` seems suboptimal. Maybe there's a good way around that. A ChainMap?